Assets/Scripts/Game/Formulas/FormulaHelper.cs

```diff
@@ -1509,9 +1509,14 @@ public static int SavingThrow(DFCareer.Elements elementType, DFCareer.EffectFlag
                 savingThrow += 25;
 
             savingThrow += biographyMod + modifier;
-            if (elementType == DFCareer.Elements.Frost && target == playerEntity && playerEntity.Race == Races.Nord)
-                savingThrow += 30;
-            else if (elementType == DFCareer.Elements.Magic && target == playerEntity && playerEntity.Race == Races.Breton)
+
+            // Apply racial resistances
+            var raceTemplate = playerEntity.GetLiveRaceTemplate();
```

Comment:
Seems like the check for whether the target is the player entity is missing.

Reply:
Yep, fixed that now.